python/gsw/main.py

```diff
@@ -0,0 +1,350 @@
+from flask import Flask, request, jsonify
+from flasgger import Swagger
+from flasgger.utils import swag_from
+from elasticsearch import Elasticsearch
+from datetime import datetime
+import threading
+import json
+import imaplib
+import base64
+import os
+import email
+import logging
+
+app = Flask(__name__)
+app.config["SWAGGER"]={"title": "PAN Ground Software", "uiversion": 2}
+ 
+class read_iridium(object):
+    def __init__(self, radio_keys_config, server_keys_config):
```

Comment:
One major feature we absolutely need is separating telemetry by IMEI number. Please consider doing this soon (this is higher priority than the EEPROM controller.)